meson.build

```diff
@@ -18,40 +18,40 @@ project(
     meson_version : '>=0.49'
 )
 
-pkg_config_name = 'libtaskomat'
+pkg_config_name = 'taskolib'
 
 # Keep this version number up to date...
-sol_version = '3.2.3'
+sol_version = '3.3.0'
```

Comment:
Bumping the Sol version should be left to the other PR, shouldn't it?

Reply:
Yes. Obvioulsy working on too many changes in parallel.